### PR TITLE
 Repo group dark mode

### LINKED ISSUE #11922: Feedback for Custom Search Pages and new Homepage Design
#### High priority bugs

- [x] 404 bug: #11961
- [x] Dark theme Kubernetes icon fix
   - Issue: #11956 
   - PR: #12038
- [x] Remove fancy quotes in message 
   - Issue: #11953 
   - [PR #12061](https://github.com/sourcegraph/sourcegraph/pull/12061#discussion_r452519930)
- [x] Lots of content fixes! #11939
- [x] Fix responsiveness for sections below list of repository groups 
   - Issue: #12040
   - PR: #12080 
   - Context:
      - #11783 (comment)
      - [Slack convo](https://sourcegraph.slack.com/archives/C0HMGV90V/p1594171649071100)
- [x] Indexing repos #11962
- [x] Border-radius on repogroup page #11955 
- [x] Fix contrast in dark theme, headings, links
   - PR: #12078
   - Design: [New styles updated in Figma ](https://www.figma.com/file/Xc4M24VTQq8itU0Lgb1Wwm/RFC-159-Visual-Design?node-id=0%3A1&viewport=-4312%2C-352%2C1)
- [x] Repogroup page copy feedback: #12057
   - PR: #12084 
- [x] Repogroup page search bar should be populated with the repogroup #12083 
- [x] [RepogroupPage should wrap responsively](https://github.com/sourcegraph/sourcegraph/pull/11783#discussion_r449064515)
- [x] Distinguish code and non-code text by font 
- Issue: [#12057](https://github.com/sourcegraph/sourcegraph/issues/12057#issue-654306195)(bullet 2)
- PR: https://github.com/sourcegraph/sourcegraph/pull/12078/

#### Original PR feedback

- [x] [Represent exampleQuery as JSX expression not HTML string](https://github.com/sourcegraph/sourcegraph/pull/11783#discussion_r449047593)
- [ ] [Use all SVG logos, and use JSX expression](https://github.com/sourcegraph/sourcegraph/pull/11783#discussion_r449049081)
- [x] [Fix leading whitespace](https://github.com/sourcegraph/sourcegraph/pull/11783#discussion_r449049691)
- [ ] [Update repogroup filenames to have proper casing](https://github.com/sourcegraph/sourcegraph/pull/11783#discussion_r449050247)
- [ ] [Search page growth](https://github.com/sourcegraph/sourcegraph/pull/11783#discussion_r449056041)
- [ ] [Use classNames() for better readability](https://github.com/sourcegraph/sourcegraph/pull/11783#discussion_r449063728)
- [ ] [Color and Font related manual overrides and inconsistencies](https://github.com/sourcegraph/sourcegraph/pull/11783#discussion_r449052724)

#### Feature Suggestions

1. count:xxx or lang:xxx is repeated often, too busy visually?
2. Copy-paste ability for search syntax on page
   - feedback: "might be nice to have this text be accompanied by a copy button and/or have it prefilled in the search box :mag:"
image (3)
3. Hide/Collapse suggestions on the /search homepage if it feels busy

### PR DESCRIPTION
**Pull request by rrhyne**
Fixes #11922

Fixes contrast issues with dark mode, corrects text and link colors, segments light and dark colors from styles and other issues related to the new homepage. 

Home before:

Screen Shot 2020-07-09 at 8 34 03 PM

Home after:

Screen Shot 2020-07-09 at 8 34 16 PM

Repository group page before:

Screen Shot 2020-07-09 at 8 34 09 PM

Repository group page after:

Screen Shot 2020-07-09 at 8 34 20 PM

### CODE REVIEW

**Comment by twop** [comment]
also, thanks for working on this! <3

File: web/src/repogroups/RepogroupPage.scss [hunk 1]
```diff
@@ -102,5 +90,69 @@
...
+
+        h1,
+        h2 {
+            color: $gray-17;
+        }
+
+        h3,
+        h4,
+        h6 {
+            color: $color-border-hover;
```
Comment (felixfbecker) [inline web/src/repogroups/RepogroupPage.scss:102]: Using this variable for a text color doesn't seem right to me. If we wanted to update only what borders look like and change this variable, wouldn't it be unexpected that these headings change too?

File: web/src/search/input/SearchPage.scss [hunk 2]
```diff
@@ -15,6 +15,21 @@
     overflow: auto;
     padding-bottom: 4rem;
 
+    &__web-link {
+        color: #1c7cd6;
+    }
+
+    h5 {
+        text-transform: uppercase;
```
Comment (felixfbecker) [inline web/src/search/input/SearchPage.scss:23]: Isn't this already on the global `h5`?

https://github.com/sourcegraph/sourcegraph/blob/cab1a64cfdbe0c930cccdc9fd550e60913972b60/web/src/global-styles/type.scss#L32

File: web/src/repogroups/RepogroupPage.scss [hunk 3]
```diff
@@ -102,5 +90,69 @@
...
+
+.theme-dark {
+    .repogroup-page {
+        h1,
+        h2 {
+            color: $gray-05;
+        }
+
+        h3,
+        span.h3,
```
Comment (felixfbecker) [inline web/src/repogroups/RepogroupPage.scss:131]: This looks odd - do we actually use that anywhere? If yes, I'd make it just `.h3`, as it's odd to require it to be a `span`

**Pull request merged**